Documentation/operations/troubleshooting.rst

```diff
@@ -713,6 +713,38 @@ When running in :ref:`arch_direct_routing` mode:
 
 4. Verify that the firewall on each node permits to route the endpoint IPs.
 
+Traffic to Pods of a Service is unbalanced
+------------------------------------------
+
+Symptom
+~~~~~~~
+
+Endpoint to endpoint communication between nodes is successful, but connections
+and / or load appears unbalanced to one or more Pods that make up a Service.
+One or more Pods receives more traffic than the others.
+
+Troubleshooting steps:
+~~~~~~~~~~~~~~~~~~~~~~
+
+Confirm the load balancing mode enabled for Services in the cluster, set by the
+``hostServices.enabled`` Helm flag or the ``enable-host-reachable-services``
+flag passed to the Cilium agent. More information found on Host-Reachable
+services in :ref:`host-services`. Another option is to consider whether running
+without ``kube-proxy`` is an option for your cluster. This feature makes use of
+the host-reachable services feature, above. More details can be found in
+:ref:`kubeproxy-free`, along with the requirements, and steps for enabling.
+
+If host-reachable services are disabled, (i.e.
+``hostServices.enabled=false``), the following parameters can provide more
+fine-grained control over service load balancing in the cluster:
+
+  * ``bpf-ct-timeout-service-*`` - The amount of time endpoints are tracked via
```

Comment:
We have two types of parameters in this section - helm vars and configmap vals. I'd stick to one type to avoid confusion.

Reply:
Also, could you mention full flag names (TCP and other L4 protos)?